### PR TITLE
Config file bug fixes.

### PR DESCRIPTION
**Pull request by prescod**

### CODE REVIEW

File: config/settings/integration_tests.py [hunk 1]
```diff
@@ -61,9 +61,15 @@
...
-ARTICLE_BODY_CLASS = env('ARTICLE_BODY_CLASS', "sfdo-kb__body")
+ARTICLE_AUTHOR = env('ARTICLE_AUTHOR', default = "ArticleOwner")
+ARTICLE_AUTHOR_OVERRIDE = env('ARTICLE_AUTHOR_OVERRIDE', default = "ArticleContributorOverride")
+ARTICLE_BODY_CLASS = env('ARTICLE_BODY_CLASS', default = "sfdo-kb__body")
+
+# SECRET CONFIGURATION
+# ------------------------------------------------------------------------------
+# See: https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
+# Note: This key only used for development and testing.
+SECRET_KEY = env('DJANGO_SECRET_KEY', default='RPr%].dHP4VLcU}SW(5o4;cH(]4i7?noV.q5*.%16!@#TYO/ku')
```
Comment (davisagli) [inline config/settings/integration_tests.py]: I gueeessss this is okay in this case...

File: config/settings/integration_tests.py [hunk 2]
```diff
@@ -61,9 +61,15 @@
 # ------------------------------------------------------------------------------
 
 # article related
-ARTICLE_AUTHOR = env('ARTICLE_AUTHOR', "ArticleOwner")
-ARTICLE_AUTHOR_OVERRIDE = env('ARTICLE_AUTHOR_OVERRIDE', "ArticleContributorOverride")
-ARTICLE_BODY_CLASS = env('ARTICLE_BODY_CLASS', "sfdo-kb__body")
+ARTICLE_AUTHOR = env('ARTICLE_AUTHOR', default = "ArticleOwner")
+ARTICLE_AUTHOR_OVERRIDE = env('ARTICLE_AUTHOR_OVERRIDE', default = "ArticleContributorOverride")
+ARTICLE_BODY_CLASS = env('ARTICLE_BODY_CLASS', default = "sfdo-kb__body")
```
Comment (davisagli) [inline config/settings/integration_tests.py]: Did you run black? Keyword arguments shouldn't have spaces around the equal sign, and the quotes are inconsistent.

File: config/settings/integration_tests.py (hunk 1 above)
Comment (prescod) [reply config/settings/integration_tests.py]: I had actually deleted it before (that's why I had to re-introduce it) but then I realized that 

a) it is kind of silly to delete a key that is just a cut and paste from another part of the project (produciton.py)

b) its an auto-generated key that Django puts in these files.

c) its only used for development and testing.

d) the code doesn't work without it and I shouldn't make it unnecessarily hard for other developers to get started.

File: config/settings/integration_tests.py (hunk 2 above)
Comment (prescod) [reply config/settings/integration_tests.py]: Black wanted to change basically every line of the file and I thought that keeping it "diff-friendly" with the other config file was a higher priority. I guess I could make a commit blackifying that whole directory later.

But I fixed the thing you noticed.

Comment (davisagli) [reply config/settings/integration_tests.py]: Okay, doing a separate blackening sounds good.

**Pull request merged**